tools/m1_utils/update_dynamic.sh

```diff
@@ -19,10 +19,12 @@ patch() {
      fi
 
      # FIXME: Versions should be input from the build system
-     xcrun vtool -arch arm64 -set-build-version 7 11.0 11.0 -replace -output "$OF" "OF.ar"
+     # stderr is ignored to silence expected warnings that the old code signatures will be invalid
+     xcrun vtool -arch arm64 -set-build-version 7 11.0 11.0 -replace -output "$OF" "OF.ar" 2> /dev/null
```

Comment:
( may need a robust version of checking for a sig w/o verifying )

Reply:
Good idea, I wasn't sure if other useful output would come through stderr for these commands.  I'll give this a try tomorrow

Reply:
The signature was removed successfully but seems to have mangled the file somehow:

```
/Applications/Xcode-14.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip: fatal error: file not in an order that can be processed (link edit information does not fill the __LINKEDIT segment)
```

Reply:
If `codesign --remove-signature` is too messed up,  WDTY to just pipe stderrr and `grep --invert $CODE_SIGNATURE_ERROR`, otherwise conditionally swallowing for signed ones only..

Reply:
Otherwise,  `grep --invert "warning:"`🙈